repo/packages/C/chronos/0/marathon.json

```diff
@@ -28,6 +28,8 @@
       "portIndex": 0,
       "protocol": "HTTP"
     }
-  ]
-
+  ],
+  "labels": {
+    "DCOS_PACKAGE_FRAMEWORK_NAME": "{{chronos/mesos-framework-name}}"
```

Comment:
Yes, ideally Cassandra shouldn't be doing that either. Unfortunately currently Cassandra generate the framework name from other information. In this case the cluster name.

Having said that, this is not the case for Chronos. Chronos doesn't generate the framework name. It is instead configurable in a very similar fashion to Marathon. We should use the same mechanism that Marathon uses.